parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -292,6 +292,11 @@ write_files:
                 "subnet": "{{`{{.PodCIDR}}`}}",
                 "routes": [{ "dst": "0.0.0.0/0" }]
             }
+          },
+          {
+            "type": "portmap",
+            "capabilities": {"portMappings": true},
+            "snat": false
```

Comment:
any particular reason we need to disable snat?

Reply:
Changed to use existing KUBE-MARK-MASQ bitmask for SNAT.

Essentially this rule will have traffic from localhost(127.0.0.1) and the podIP(i.e 10.244.x.x) of be SNAT'ed back to the node IP. Docker clusters have a similar set of rules.